lib/process-services/task-list/components/attach-form.component.spec.ts

```diff
@@ -54,7 +54,7 @@ describe('AttachFormComponent', () => {
 
     it('should call attachFormToATask if clicked on Complete Button', async(() => {
         component.taskId = 1;
-        component.formKey = 2;
```

Comment:
can we have a test related to the remove and attached form?

Reply:
another test could be if a task has a form we should check that the remove button is there

Reply:
another related to the change form

I have a task with a form
I select another form
I do attach
expected the task shoud have the new form